src/main/resources/data.sql

```diff
@@ -1,42 +1,14 @@
-SET session_replication_role = replica;
+INSERT INTO ROLE(id, authority)
+VALUES (1, 'ADMIN'),
+       (2, 'USER');
 
 insert into USERS (username, first_name, last_name, email, password, role_id, was_released)
 values ('wichni', 'Jakub', 'Wichniarek', 'wichniarekjakub89@gmail.com', '123456', 1, false ),
-        ('przydan', 'Patryk', 'Przydanek', 'przydan@hotmail.com','123456', 1, false ),
```

Comment:
czemu to usuwasz?

Reply:
users i roles się nie zmieniły , resztę muszę poprawić